src/index.html

```diff
@@ -11,12 +11,99 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__block">
```

Comment:
i would not like to write style straight to header , to my mind it will not be the best decision. I understand that it is a very small project but still , what do you think or is it supper necessary ?

Reply:
Please try remove this block, think it critical not needed